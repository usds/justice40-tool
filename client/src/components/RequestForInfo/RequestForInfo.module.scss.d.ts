declare namespace RequestForInfoNamespace {
    export interface IRequestForInfoScss {
        rfiBox: string;
    }
  }

declare const RequestForInfoScssModule: RequestForInfoNamespace.IRequestForInfoScss & {
    /** WARNING: Only available when `css-loader` is used without `style-loader` or `mini-css-extract-plugin` */
    locals: RequestForInfoNamespace.IRequestForInfoScss;
  };

  export = RequestForInfoScssModule;
