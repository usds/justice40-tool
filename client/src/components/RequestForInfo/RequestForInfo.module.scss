@use '../../styles/design-system.scss' as *;

.rfiBox {
    @include at-media-max("desktop") {
        @include u-margin-top(2);
        @include u-margin-bottom(2);

    };
    
    @include u-margin-top(10);

}