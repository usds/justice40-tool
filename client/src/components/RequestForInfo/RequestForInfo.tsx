import React from 'react';
import {SummaryBox, SummaryBoxContent, SummaryBoxHeading} from '@trussworks/react-uswds';
import {useIntl} from 'gatsby-plugin-intl';

import * as CONTACT_COPY from '../../data/copy/contact';
import * as styles from './RequestForInfo.module.scss';

const RequestForInfo = () => {
  const intl = useIntl();

  return (
    <SummaryBox className={styles.rfiBox}>

      <SummaryBoxHeading headingLevel='h3'>
        {intl.formatMessage(CONTACT_COPY.RFI_BOX.TITLE)}
      </SummaryBoxHeading>

      <SummaryBoxContent>
        <p>{CONTACT_COPY.RFI_BOX_BODY}</p>
      </SummaryBoxContent>

    </SummaryBox>
  );
};

export default RequestForInfo;
