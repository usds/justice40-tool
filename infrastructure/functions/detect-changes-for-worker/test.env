REGION=us-east-1
STAGE=sit

# ESC Cluster name that will from the containers
ECS_CLUSTER=j40-sit-justice40-data-harvester-ECSCluster-ktXGGU9zjwkb

# VPC Private Subnet that has a NAT GAteway
VPC_SUBNET_ID=subnet-07e68cb57322f7b1f

# Names of the container and task names.
GDAL_TASK_DEFINITION=sit-justice40-data-harvester-gdal
GDAL_CONTAINER_DEFINITION=sit-justice40-data-harvester-osgeo-gdal
TIPPECANOE_TASK_DEFINITION=sit-justice40-data-harvester-tippecanoe
TIPPECANOE_CONTAINER_DEFINITION=sit-justice40-data-harvester-mgiddens-tippecanoe

# AWS Credentials
AWS_ACCESS_KEY_ID=
AWS_SECRET_ACCESS_KEY=
AWS_DEFAULT_REGION=us-east-1