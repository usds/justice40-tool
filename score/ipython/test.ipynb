{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "1a4c0c68",
   "metadata": {},
   "outputs": [],
   "source": [
    "import numpy as np\n",
    "import pandas as pd"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 49,
   "id": "70b3a793",
   "metadata": {},
   "outputs": [],
   "source": [
    "df = pd.read_csv('data/fips_states.csv')  "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 51,
   "id": "c514aad8",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>fips</th>\n",
       "      <th>state_name</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>1</td>\n",
       "      <td>Alabama</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>2</td>\n",
       "      <td>Alaska</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>4</td>\n",
       "      <td>Arizona</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>5</td>\n",
       "      <td>Arkansas</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>6</td>\n",
       "      <td>California</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "   fips  state_name\n",
       "0     1    Alabama \n",
       "1     2     Alaska \n",
       "2     4    Arizona \n",
       "3     5   Arkansas \n",
       "4     6  California"
      ]
     },
     "execution_count": 51,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "b9ee44d9",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.0"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
