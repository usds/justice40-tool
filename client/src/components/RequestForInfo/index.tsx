import RequestForInfo from './RequestForInfo';

export default RequestForInfo;
