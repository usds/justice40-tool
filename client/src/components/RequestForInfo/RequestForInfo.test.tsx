import * as React from 'react';
import {render} from '@testing-library/react';
import {LocalizedComponent} from '../../test/testHelpers';
import RequestForInfo from './RequestForInfo';

describe('rendering of the RequestForInfo', () => {
  const {asFragment} = render(
      <LocalizedComponent>
        <RequestForInfo/>
      </LocalizedComponent>,
  );

  it('checks if component renders', () => {
    expect(asFragment()).toMatchSnapshot();
  });
});
