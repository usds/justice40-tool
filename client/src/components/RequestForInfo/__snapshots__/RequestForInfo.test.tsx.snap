// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rendering of the RequestForInfo checks if component renders 1`] = `
<DocumentFragment>
  <div
    class="usa-summary-box"
    data-testid="summary-box"
  >
    <div
      class="usa-summary-box__body"
    >
      <h3
        class="usa-summary-box__heading"
      >
        Request for Information
      </h3>
      <div
        class="usa-summary-box__text"
      >
        <p>
          The Request for Information on the Federal Register for the public beta period closed on May 25, 2022.
        </p>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
